packages/virtualized-lists/Lists/VirtualizedList.js

```diff
@@ -1552,16 +1552,16 @@ class VirtualizedList extends StateSafePureComponent<Props, State> {
       onStartReached != null &&
       this.state.cellsAroundViewport.first === 0 &&
       isWithinStartThreshold &&
-      this._listMetrics.getContentLength() !== this._sentStartForContentLength
+      this.state.firstVisibleItemKey !== this._sentStartForFirstVisibleItemKey
```

Comment:
Comment above is now out of date it looks like

Reply:
Fixed, `and only if onEndReached is not being executed` that part was also out of date from a previous change.